WORKSPACE

```diff
@@ -47,6 +47,17 @@ python_deps()
 load("@vaticle_dependencies//builder/antlr:deps.bzl", antlr_deps = "deps", "antlr_version")
 antlr_deps()
 
+# Load //builder/rust
+load("@vaticle_dependencies//builder/rust:deps.bzl", rust_deps = "deps")
+rust_deps()
+
+load("@rules_rust//rust:repositories.bzl", "rust_repositories")
+rust_repositories(version = "nightly", iso_date = "2021-07-01", edition="2018")
```

Comment:
Nightly version needs to be used as ANTLR-generated code uses `#![feature]`s that are not available in stable release (yet)